Update CHANGELOG for 1.0.8 release (#1477)
bottlerocket-os/bottlerocket: CHANGELOG.md

# v1.0.8 (2021-04-12)

## Deprecation Notice

> Bottlerocket 1.0.8 is the last release where we plan to support the Kubernetes 1.15 variant, `aws-k8s-15`.  Kubernetes 1.15 is no longer receiving support upstream.  We recommend replacing `aws-k8s-1.15` nodes with a later variant, preferably `aws-k8s-1.19` if your cluster supports it.

Reviewer: > `aws-k8s-15`.

nit: typo, `aws-k8s-1.15`

> Bottlerocket 1.0.8 is the last release where we plan to support the Kubernetes 1.15 variant, `aws-k8s-1.15`.  Kubernetes 1.15 is no longer receiving support upstream.  We recommend replacing `aws-k8s-1.15` nodes with a later variant, preferably `aws-k8s-1.19` if your cluster supports it.

Reviewer 2: Can we link to the deprecation tracking issue #1478

# v1.0.8 (2021-04-12)

## Deprecation Notice

> Bottlerocket 1.0.8 is the last release where we plan to support the Kubernetes 1.15 variant, `aws-k8s-1.15`.

Reviewer 3: I'd prefer to say:
```suggestion
Bottlerocket 1.0.8 is the last planned release that will support the Kubernetes 1.15 variant, `aws-k8s-1.15`.
```

Since we may do unplanned releases with critical security or bug fixes prior to the EOL in early May.

Reviewer 3: Discussed out-of-band - it seems like we are trying to express the same thing, so I'm fine with this wording and we can adjust it if it causes confusion.

* Update default host container versions ([#1443], [#1441], [#1466])
* Fix avc denial for dbus-broker ([#1434])
* Fix case of outputted JSON keys in host container user data ([#1439])
* Set mode of host container persistent storage directory after creation ([#1463])
* Add "current" persistent storage location for host containers ([#1416])
* Write static-pods manifest to tempfile before persisting it ([#1409])

## Build Changes

* Update default variant to aws-k8s-1.19 ([#1394])
* Update third-party packages ([#1460])
* Update Rust dependencies ([#1461], [#1462])
* Update dependencies of host-ctr ([#1371])
* Add support for specifying a variant's supported architectures ([#1431])
* Build OVA packages and include them in repos ([#1428])
* Add support for qcow2 as an image format ([#1425]) (Thanks, @mikalstill!)
* Prevent unneeded artifacts from being copied through build process ([#1426])
* Change image format for vmware-dev variant to vmdk ([#1397])
* Remove tough dependency from update_metadata ([#1390])
* Remove generate_constants logic from build.rs of parse-datetime ([#1376])
* In the tools workspace, update to tokio v1, reqwest v0.11, and tough v0.11 ([#1370])

> * Don't include metrics-related default settings in vmware-dev ([#1365])

Reviewer 3: I'd rather omit this note since we're about to add them back in #1452.
